notes/ssh-keys.md

```diff
@@ -0,0 +1,21 @@
+### ssh keys
+
+create ssh key pair from workstation -> server
+
+create ssh key pair from server -> github
+
+[https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent]
+
+`ssh-copy-id -i ~/.ssh/id_ed25519.pub stephen@192.168.86.22`
```

Comment:
should this be `ssh-copy-id -i ~/.ssh/id_ed25519.pub <username>@<rasp-pi IP addr>`

Reply:
Yeah maybe, but those are just notes. We need to work those notes into the formal README after provisioning. We'll do that next.